### ISSUE TITLE
Export this book to a more flexible format like ePub or Kindle

### ISSUE THREAD

**Issue by Birowsky** [AnomalyInnovations/serverless-stack-com#325]
Just a feature request.

**Comment by jayair** [comment]
Yeah this is definitely something we want to do. Just haven't found the time to do it. Hoping somebody is willing to contribute!

**Comment by pepas24** [comment]
Hi @Birowsky We just start working more in this improvement, please take a look to [this version](https://drive.google.com/file/d/1WUqipyj1Dqt6u-GrW_00hD9UdKUX8pC2/view?usp=sharing) of pdf and tell us what do you think about it. Still needs improves but I would appreciate your comments very much.
Thanks

**Comment by Birowsky** [comment]
@pepas24 there is not much to say other than it looks beautiful. Beautiful font, comfortable line height.  You've made all the right decisions, generally speaking. But I'm "special" : ) I want seriffed fonts, huge line height, huge fonts, no padding, endless scroll, black on white when daytime, and white on black at night :} 

Again, the PDF looks stunning, but these are the reasons I was asking for an epub. Thanx for the great work!

**Comment by pepas24** [comment]
Ok I understand your point hahah I forgot that we have an [epub version](https://drive.google.com/open?id=1xImlIMsnpeedZoM-1gTLob9o1bNdUeIz) too ;) , still in progress
I think that I answer the incorret issue.

**Comment by moltar** [comment]
Haven't tried it, but there is this: https://github.com/glejeune/jekyll-epub
